Steps.md

```diff
@@ -0,0 +1,25 @@
+Git tips
+========
+Overview
```

Comment:
**# Overview** [Esto debería ir en negrita y texto mas grande]
[agrega una linea aqui]
This steps summarize the most common git flow steps used to work with remote repositories using github.